ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -139,48 +144,62 @@ open class TwoLineTitleView: UIView, TokenizedControlInternal {
 
     @objc public weak var delegate: TwoLineTitleViewDelegate?
 
-    private var alignment: Alignment = .center
+    var currentStyle: Style {
+        didSet {
+            applyStyle()
+        }
+    }
+
+    private lazy var alignmentConstraint: NSLayoutConstraint = centerXAnchor.constraint(equalTo: containingStackView.centerXAnchor)
+    private var alignment: Alignment = .center {
+        didSet {
+            guard alignment != oldValue else {
+                return
+            }
+            alignmentConstraint.isActive = false
+            let keyPath = alignment.xAxisKeyPath
+            alignmentConstraint = self[keyPath: keyPath].constraint(equalTo: containingStackView[keyPath: keyPath])
+            alignmentConstraint.isActive = true
+        }
+    }
     private var interactivePart: InteractivePart = .none
     private var animatesWhenPressed: Bool = true
     private var accessoryType: AccessoryType = .none
 
-    private let titleButton = EasyTapButton()
-    private var titleAccessoryType: AccessoryType {
-        return interactivePart.contains(.title) ? accessoryType : .none
-    }
+    private lazy var containingStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = 0
+        return stackView
+    }()
 
-    var currentStyle: Style {
-        didSet {
-            applyStyle()
-        }
+    private static func titleLineStackView() -> UIStackView {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.spacing = TokenSetType.titleStackSpacing
+        return stackView
     }
 
-    private lazy var titleButtonLabel: Label = {
+    private lazy var titleContainer = Self.titleLineStackView()
```

Comment:
It's so we don't have to type out the same lazy initialization bit twice. And since this function doesn't involve any particulars of a single instance of `TwoLineTitleView`, there's no reason for it to *not* be static.

Reply:
I've renamed the function to make things a little bit clearer. Maybe it'll make things clearer.